zigConsole/Dockerfile.sdk

```diff
@@ -0,0 +1,34 @@
+# ARGUMENTS --------------------------------------------------------------------
+ARG CROSS_SDK_BASE_TAG=3.0.9-bookworm
+
+##
+# Board architecture
+# arm or arm64
+##
+ARG IMAGE_ARCH=
+
+
+# BUILD ------------------------------------------------------------------------
+FROM torizon/debian-cross-toolchain-${IMAGE_ARCH}:${CROSS_SDK_BASE_TAG}
+
+# __deps__
+RUN apt-get -q -y update && \
+    apt-get -q -y install \
+    # ADD YOUR PACKAGES HERE
+    && \
+    apt-get clean && apt-get autoremove && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
here you are using the `torizon/debian-cross-toolchain-<arch>` but are not installing zig, this image has only the build essentials for the architecture chosen: https://github.com/toradex/torizon-containers/blob/bookworm/debian-cross-toolchains/base/Dockerfile

if you try to run the `build-debug-arm64` for example, this will error with:

```
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "zig": executable file not found in $PATH: unknown.
```

Reply:
update: new commit on `docker-zig` add multiples architectures on `bookworm-ziglang` image:
https://github.com/kassane/docker-zig/commit/414d2851564f7f822407618c6c0989c91cacf2e0